test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Formatter/CollectionPropertyTests.cs

```diff
@@ -69,7 +70,10 @@ public async Task SupportPostCollectionPropertyByEntityPayload()
             // clear respository
             await this.ClearRepositoryAsync("CollectionProperty_Entity");
 
-            var rand = new Random(RandomSeedGenerator.GetRandomSeed());
+            var seed = RandomSeedGenerator.GetRandomSeed();
+            Trace.WriteLine($"Generated seed for random number generator: {seed}");
```

Comment:
where do you retrieve the trace information? If it's running on Azure pipeline, is it easy to trace?

Different test cases output the same trace, is it easy to identify?

Reply:
I guess I am only expecting someone to need the seed in the event of an intermittent test failure that needs to be investigated, so in the pipeline logs, when it logs the failed test, the trace will be right above the name of the failed test

Reply:
Are you thinking I should add the test name to the message or something along those lines?

Reply:
@xuzhg, I've updated the change so that we only log 1 time. It is relatively straightforward to find the seed in the build logs.